packages/js/src/integrations-page/plugin-integration.js

```diff
@@ -4,27 +4,39 @@ import { __ } from "@wordpress/i18n";
 import { PropTypes } from "prop-types";
 import { SimpleIntegration } from "./simple-integration";
 
+// Flag to check if the Schema Framework is enabled.
+// eslint-disable-next-line dot-notation
+const isSchemaFrameworkEnabled = Boolean( window.wpseoIntegrationsData[ "schema_framework_enabled" ] );
```

Comment:
We have multiple usages of this very same line, maybe there's a more graceful way to pass that information along the component tree? We can consider [React's context](https://react.dev/learn/passing-data-deeply-with-context) for this?